src/main/java/org/killbill/billing/client/KillBillClient.java

```diff
@@ -1214,19 +1214,20 @@ public Invoice adjustInvoiceItem(final InvoiceItem invoiceItem, final LocalDate
 
 
     @Deprecated
-    public InvoiceItem createExternalCharge(final InvoiceItem externalCharge, @Nullable final LocalDate requestedDate, final Boolean autoPay, final Boolean autoCommit, final String createdBy, final String reason, final String comment) throws KillBillClientException {
```

Comment:
2 thoughts:

* I would keep the signature where no key is passed because it doesn't make sense when `autoPay` is `false` (and could be confusing if the user doesn't know what to pass) - the old one could call the new one with `null` values
* No need to change / update `@Deprecated` methods (the ones without the `RequestOptions` parameter)